docs/UserGuide.md

```diff
@@ -493,19 +495,26 @@ e.g. Date of task is `2022-01-01` with `daily` recurrence. On `2022-01-03`, the
 
 </div>
 
-Example:
-* `viewTasks` followed by `editTask 1 date/2022-01-30` changes the date of the 1st task shown by NurseyBook to 30 January 2022.
+<div markdown="block" class="alert alert-warning">
+
+:exclamation: **Caution**
+* If you want to edit the elderlies' names of the task, you need to first make sure all the elderlies' names are already registered in your elderly database.
```

Comment:
Perhaps rephrasing to `If you want to edit the names of any elderlies linked to a task, ...` would be better.

Reply:
Made the changes!